src/components/text/Content.tsx

```diff
@@ -34,5 +34,4 @@ export const ContentBackgroundLight = styled(ContentBackground)`
 `
 export const ContentTextBold = styled(Content)`
   font-family: ${props => props.theme.fonts.contentFontBold};
-  font-size: ${prop => prop.theme.fonts.defaultFontSize};
```

Comment:
Please add this again. Chaning the components in `components/text/...` will change the text everywhere in the app. By my last comment, because of which you removed this, I meant you should remove the `font-size`-Attribute from your styled component that you have created, as if you use

```
const MyComponent = styled(Conent)'
 font-size = prop.theme.fonts.defaultFontSize
'
```

the font-size here is just not needed as `MyComponent` just extends `Content` and therefore all attributes of Content (like the font-size) are already applied to MyComponent

Reply:
Oh .. my bad 😥